### PR TITLE
fix(vast): do not validate the vast 3.0 item

### PR DESCRIPTION
**Pull request by xiaopeng0216**
# Context
This VAST package is mixture support VAST 2.0 and VAST 3.0. And it is not a good approach. Eventually, we'd better to use separated package for different version support.

In this PR, I'd like to remove the VAST 3.0 validation on VAST 2.0

### CODE REVIEW

File: vast/inline.go [hunk 1]
```diff
@@ -28,14 +28,7 @@ func (inline *InLine) Validate() error {
...
 
-	if inline.Pricing != nil {
-		if err := inline.Pricing.Validate(); err != nil {
-			ve, ok := err.(ValidationError)
-			if ok {
-				errors = append(errors, ve.Errs...)
-			}
-		}
-	}
+	// No necessary to validate pricing which is supported by VAST 3.0.
```
Comment (Fan-Xu) [inline vast/inline.go]: actually the price validate did nothing, do we still need to remove it?

Comment (xiaopeng0216) [reply vast/inline.go]: Yes, I'd like to remove it since it is VAST 3.0

File: vast/companionads.go [hunk 2]
```diff
@@ -10,14 +10,7 @@ type CompanionAds struct {
...
 
-	if len(companionAds.Required) != 0 {
-		if err := companionAds.Required.Validate(); err != nil {
-			ve, ok := err.(ValidationError)
-			if ok {
-				errors = append(errors, ve.Errs...)
-			}
-		}
-	}
+	// No necessary to validate Required which for VAST 3.0 only.
```
Comment (yingchen0706v) [inline vast/companionads.go]: // No need to validate Required which for VAST 3.0 only.

Comment (Fan-Xu) [reply vast/companionads.go]: // No need to validate Required which is for VAST 3.0 only.

**Comment by Fan-Xu** [comment]
LGTM

**Pull request merged**